docs/topics/functions.md

```diff
@@ -127,37 +121,32 @@ reformat(
 )
 ```
 
-You can skip all the ones with default values:
+You can disregard all of the ones with default values:
 
 ```kotlin
 reformat("This is a long String!")
 ```
 
-You are also able to skip specific arguments with default values, rather than omitting them all. However, after the first
-skipped argument, you must name all subsequent arguments:
+You can also skip specific arguments with default values rather than omitting all of them. After the first skipped argument, however, you must name all subsequent arguments:
 
 ```kotlin
 reformat("This is a short String!", upperCaseFirstLetter = false, wordSeparator = '_')
 ```
 
-You can pass a [variable number of arguments (`vararg`)](#variable-number-of-arguments-varargs) with names using the
-`spread` operator:
+Using the'spread' operator, you can pass a [variable number of arguments ('vararg')](#variable-number-of-arguments-varargs) with names:
 
 ```kotlin
 fun foo(vararg strings: String) { /*...*/ }
 
 foo(strings = *arrayOf("a", "b", "c"))
 ```
 
-> On the JVM: You can't use the named argument syntax when calling Java functions because Java bytecode does not
-> always preserve the names of function parameters.
->
-{type="note"}
+When calling Java functions on the JVM, you cannot use the named argument syntax because Java bytecode does not always preserve the names of function parameters.
```

Comment:
```suggestion
> When calling Java functions on the JVM, you can't use the named argument syntax because Java bytecode does not always preserve the names of function parameters.
{type="note"}
```
It's important that the reader notices this, so we should keep it as a note.